### DISCUSSION TITLE
Would it be possible to use this project as a WebAssembly library?

### DISCUSSION THREAD

**Discussion by vc0d35** [coqui-ai/STT#2281] [Q&A] [answered]
Hi!

I have little C++ knowledge (for now) and I'm looking for a purely client-side-based STT solution. I stumbled upon coqui and I was wondering if it would be possible to use it as a WebAssembly library.

Thanks!

**Comment by HarikalarKutusu** [comment, marked as answer]
This has been freshly added :) It will in the next release.

https://github.com/coqui-ai/STT-examples/tree/r1.0/wasm
https://stt.readthedocs.io/en/latest/NodeJS-API.html
https://github.com/coqui-ai/STT/pulls?q=is%3Apr+is%3Aclosed
